libs/antsi/src/lib.rs

```diff
@@ -1,14 +1,500 @@
-pub fn add(left: usize, right: usize) -> usize {
-    left + right
+// Good reference to begin with: https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797
+#![no_std]
+#![cfg_attr(
+    nightly,
+    feature(provide_any, error_in_core, error_generic_member_access)
+)]
+#![cfg_attr(all(doc, nightly), feature(doc_auto_cfg))]
+#![warn(
```

Comment:
Could you copy over the `.config/config.toml` from `apps/hash-graph` instead?

Reply:
deer and error-stack both use this, to stay consistent I chose this approach instead. We might want to change that for all of them/

Reply:
Yes, the plan is to move it to the repository root. For now, it's only in `apps/hash-graph` as this is the most important crate we have currently. We could add it to `/libs` in another PR for now and block this PR on that

Reply:
fixed in [33fbd97](https://github.com/hashintel/hash/pull/1995/commits/33fbd97888e967ed368cbdfc9487de030a6be70e)